driver/driver_test.go

```diff
@@ -861,3 +755,30 @@ func createContrailNetwork(c *controller.Controller) *types.VirtualNetwork {
 	return controller.CreateMockedNetworkWithSubnet(
 		c.ApiClient, networkName, subnetCIDR, project)
 }
+
+func deleteTheOnlyHNSEndpoint(d *ContrailDriver) {
+	_, hnsEndpointID := getTheOnlyHNSEndpoint(d)
+	err := hns.DeleteHNSEndpoint(hnsEndpointID)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func getTheOnlyHNSEndpoint(d *ContrailDriver) (*hcsshim.HNSEndpoint, string) {
```

Comment:
If there are two or more HNS endpoints, this function should fail according to the name. Right now it returns the first element to matter what.

Either change the name to `getTheFirstHNSEndpoint` or return an error if there are more than one container.